lib/iris/fileformats/rules.py

```diff
@@ -796,7 +799,14 @@ def load_cubes(filenames, user_callback, loader, filter_function=None):
             if filter_function is not None and not filter_function(field):
                 continue
             # Convert the field to a Cube.
-            cube, factories, references = _make_cube(field, loader.converter)
+            try:
+                cube, factories, references = _make_cube(field,
+                                                         loader.converter)
+            except iris.exceptions.TranslationError as e:
+                if loader.fault_tolerant:
+                    skipped_fields += 1
```

Comment:
I would like to see 'skipped_fields' as a container of error messages.  We can always get a length from a collection.  There is extras value in having the collection of error messages from the loader to help the user understand what has been skipped

I am not sure of a good way to make this available to the user, but if it could be done neatly then I'd be really pleased